caffe2/CMakeLists.txt

```diff
@@ -1132,6 +1132,18 @@ if(BUILD_MOBILE_BENCHMARK)
   endforeach()
 endif()
 
+if(BUILD_MOBILE_TEST)
+  foreach(test_src ${ATen_MOBILE_TEST_SRCS})
```

Comment:
Does this need to be a separate thing?  How come this can't be part of the normal `BUILD_TEST`?

Reply:
Mainly so that we can build only tests we specifically care for mobile instead of building all the tests during mobile build. But I am ok merging the two if you feel thats better.

Reply:
I would merge them.  If they're taking too long to build, maybe we could look at adding support for a `-D` flag that would specify a keeplist that would only build a subset of tests.

Reply:
Actually I just realized, BUILD_TEST is hidden behind `NOT INTERN_BUILD_MOBILE`, https://github.com/pytorch/pytorch/blob/master/caffe2/CMakeLists.txt#L678. That is why I added this separate one. With all the different diffs I keep forgetting that I already tried looking into it before.